### PR TITLE
OpenMRS Patient Example and Mappings

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/FM2-490

Waiting on this  [PR](https://github.com/openmrs/openmrs-contrib-fhir2-ig/pull/14)  to get merged  then I can proceed with this pr.

### CODE REVIEW

File: input/fsh/omrs-patient.fsh [hunk 1]
```diff
@@ -33,3 +33,43 @@ Id:           omrs-patient-identifier-location
 Title:        "OpenMRS Patient Identifier Location"
 Description:  "OpenMRS location for which this identifier is valid"
 * value[x] only Reference(OMRSLocation)
+
+// OpenMRS Patient  Example
+
+Instance: example-openmrs-Patient
```
Comment (pmanko) [inline input/fsh/omrs-patient.fsh:39]: This definition is causing the following error in the CI: 

```
Sushi: error Element Patient.identifier.extension has minimum cardinality 1 but occurs 0 time(s). (00:05.354)
Sushi: File: /github/workspace/input/fsh/omrs-patient.fsh                        (00:05.355)
Sushi: Line: 39 - 58                                                             (00:05.358)
Sushi: error Element Patient.identifier.extension:location has minimum cardinality 1 but occurs 0 time(s). (00:05.359)
Sushi: File: /github/workspace/input/fsh/omrs-patient.fsh                        (00:05.361)
```

@sherrif10 you might need to add a location extension to the identifier definition.

**Review by pmanko — CHANGES_REQUESTED** [review]
There's a location-related error that needs a small fix - see the comment above

File: input/fsh/omrs-patient.fsh (hunk 1 above)
Comment (sherrif10) [reply input/fsh/omrs-patient.fsh:39]: Sure thank you, i updated it with location extension.

**Comment by sherrif10** [comment]
Hello @pmanko , @ibacher  @mozzy11 , hope this looks okay  Only that i need to add even those resources that we dont support as suggested by @ibacher  and all will be well.

**Pull request merged**